content/applications/finance/accounting/fiscal_localizations/localizations/india.rst

```diff
@@ -0,0 +1,214 @@
+
+.. image:: india/processing.png
+   :align: center
+   :alt: Click on Process Now to submit the invoice immediately
+
+Odoo will automatically submit the JSON Signed file to the government portal. But, if you want to process the invoice immediately, you can process it by clicking on :guilabel:`Process Now`.
+
+.. note::
+   You can find the JSON Signed file in the attachment.
+
+You can check the status of EDI with web-service of invoice under the :menuselection:`EDI Document` tab or the :guilabel:`Electronic invoicing` field.
+
+Invoice PDF Report
+~~~~~~~~~~~~~~~~~~
+
+After the invoice is submitted and validated, the invoice PDF report can be printed. The report includes IRN, acknowledgment number and date, and QR code, indicating the invoice is a valid fiscal document.
+
+.. image:: india/invoice-report.png
+   :align: center
+   :alt: IRN and QR code
+
+EDI Cancellation
+~~~~~~~~~~~~~~~~
+
+To cancel E-invoice set cancel reason and remarks at *Other info* tab in invoice.
+
+.. image:: india/e-invoice-cancellation.png
+   :align: center
+   :alt: cancel reason and remarks
+
+After clicking on **Request EDI cancellation**, the status of the **Electronic invoicing** field
+will be *To Cancel*.
+
+.. image:: india/request-cancellation.png
+   :align: center
+   :alt: request for edi cancellation
+
+.. note::
+   If you want to abort cancellation before processing the invoice, then click on *Call Off EDI Cancellation*
+
+Once you request for canceling the e-invoice, Odoo will automatically submit the JSON Signed file to the government portal. But, if you want to process the invoice immediately, you can process it by clicking on :guilabel:`Process Now`.
+
+
+Others
+------
+
+.. _india/verify-e-invoice:
+
+Verify E-invoice
+~~~~~~~~~~~~~~~~
+
+After submitting E-invoice, you can also verify the signed invoice from the government portal.
+
+#. Download the json file from the attachment.
+
+#. Open E-invoice portal: https://einvoice1.gst.gov.in/
+   go to :menuselection:`Search --> Verify Signed Invoice`
+
+#. Browse json file and submit
+
+   .. image:: india/verify-invoice.png
+      :align: center
+      :alt: browse json file for verify invoice
+
+#. you can check verified signed E-invoice here
+
+   .. image:: india/signed-invoice.png
+      :align: center
+      :alt: verified e-invoice
+
+.. _india/e-invoice-api_registration:
+
+E-Invoice API Registration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Login to NIC E-Invoice Web Portal on - https://einvoice1.gst.gov.in/
+
+   .. note::
+      If you have already registered on NIC Eway Bill Production portal then you may use the same Login credentials here on - https://einvoice1.gst.gov.in/
+
+#. Click on Login and enter your Username and Password
+
+   .. image:: india/e-invoice-system-login.png
+      :align: center
+      :alt: e-invoice system login
+
+#. You will be able to see similar screen after login. Now, click on :guilabel:`API Registration`
```

Comment:
```suggestion
#. After logging in, you are presented with this screen. Now, click on :guilabel:`API Registration`
```
We try to avoid the use of the future tense. Suggestion would be better,